### DISCUSSION TITLE
How we add SSH keys in Github. I am using windows 11

### DISCUSSION THREAD

**Discussion by uchiha11111** [community/community#132531] [Discussions]
How we add SSH keys in Github. I am using windows 11.

**Comment by devmuse7** [comment]
You can refer: https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account
I'll suggest you to read Docs and dig a little more.

**Reply by LiteBrite82** [reply]
Hi @Itachii27 

We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. 

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
